assignments/arrays.js

```diff
@@ -63,32 +63,44 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+let carInfo = inventory.find(nameKey => nameKey.id === 33);
```

Comment:
perfect way to get this done. i would suggest using more semantic naming for your callback functions, since every element in the inventory array represents a car, you could say >>

```let carInfo = inventory.find(car => car.id === 33);```